### ISSUE TITLE
[HOLD for payment 2024-08-05] [$250] Expensify Card - Offline indicator is below the content in "Choose a card type" page

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#45272]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.6-1
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Launch New Expensify app
2. Go to workspace settings > Expensify Card (enable in More features)
3. Go offline
4. Tap Issue new card
4. Proceed to Step 2 - Choose a card type

## Expected Result:
The offline indicator will be at the bottom of the screen

## Actual Result:
The offline indicator is below the content

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/146bf5a5-6842-4974-8a81-6c6f7ffc67c4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0101efe70a56889c26</li>
        <li>Upwork Job ID: 1812802148584923724</li>
        <li>Last Price Increase: 2024-07-15</li>
<li>Automatic offers: </li>
<ul>
<li>DylanDylann | Reviewer | 103213878</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @DylanDylann</details>

**Comment by lanitochka17** [comment]
@Gonals  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by lanitochka17** [comment]
We think that this bug might be related to #wave-collect - Release 2

**Comment by etCoderDysto** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Offline indicator is below the content in "Choose a card type"
### What is the root cause of that problem?
We are not adding `offlineIndicatorStyle={styles.mtAuto}` to ScreenWrapper here
https://github.com/Expensify/App/blob/105dce5151053bf6c81e60e68a610dd888c5da45/src/pages/workspace/card/issueNew/CardTypeStep.tsx#L44-L47
### What changes do you think we should make in order to solve the problem?
Add `offlineIndicatorStyle={styles.mtAuto}` to ScreenWrapper here

### What alternative solutions did you explore? (Optional)

**Comment by etCoderDysto** [comment]
I can raise a quick PR.

**Comment by Julesssss** [comment]
Not a release blocker, demoting.

**Comment by DylanDylann** [comment]
@etCoderDysto's [proposal](https://github.com/Expensify/App/issues/45272#issuecomment-2222918196) looks good to me


🎀 👀 🎀 C+ reviewed

**Comment by etCoderDysto** [comment]
@DylanDylann [PR](https://github.com/Expensify/App/pull/45908) is ready for review.

**Comment by sakluger** [comment]
Summarizing payment on this issue:

Contributor: @etCoderDysto $250, sent offer via Upwork ([Offer](https://www.upwork.com/nx/wm/offer/103409705))
Contributor+: @DylanDylann $250, paid via Upwork

@etCoderDysto please let me know once you've accepted the offer.
@DylanDylann could you please complete the BZ checklist?

**Comment by DylanDylann** [comment]
BugZero Checklist: The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
 
 [@DylanDylann] The PR that introduced the bug has been identified. Link to the PR: **NA**
 [@DylanDylann] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: **NA**
 [@DylanDylann] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: **NA**
 [@DylanDylann] Determine if we should create a regression test for this bug. **Yes**
 [@DylanDylann] If we decide to create a [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) for the bug, please propose the regression test steps to ensure the same bug will not reach production again.


Regression Test Proposal

1. Go to workspace settings > Expensify Card (enable in More features)
2. Go offline
3. Tap Issue new card
4. Select the added bank account
5. Proceed to Step 2 - Choose a card type
6. Verify that offline indicator is at the bottom of the screen (on narrow screen)

Do we agree 👍 or 👎

**Comment by sakluger** [comment]
Thanks everyone!
